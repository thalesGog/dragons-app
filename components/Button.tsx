import styled from 'styled-components'

const Button = styled.button`
  padding: 0.75rem 1.25rem;
  font-size: 1.25rem;
  line-height: 1.33333;
  min-width: 10rem;
  color: #fff;
  background-color: #6400e4;
  border-color: transparent;
  border: 2px solid transparent;
  transition: all 0.4s;
  cursor: pointer;
  &:hover {
    color: #6400e4;
    background-color: transparent;
    border: 2px solid #6400e4;
  }
`

export default Button
