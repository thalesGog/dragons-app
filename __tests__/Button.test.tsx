import React from 'react'
import {render} from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import Button from '../components/Button'

test('Render Button component', async () => {
  render(<Button />)
})
