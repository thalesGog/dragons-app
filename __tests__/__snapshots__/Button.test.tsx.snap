// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button component 1`] = `
.c0 {
  padding: 0.75rem 1.25rem;
  font-size: 1.25rem;
  line-height: 1.33333;
  min-width: 10rem;
  color: #fff;
  background-color: #6400e4;
  border-color: transparent;
  border: 2px solid transparent;
  -webkit-transition: all 0.4s;
  transition: all 0.4s;
  cursor: pointer;
}

.c0:hover {
  color: #6400e4;
  background-color: transparent;
  border: 2px solid #6400e4;
}

<div>
  <button
    class="c0"
  />
</div>
`;
